networkit/test/test_clique.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+import unittest
+import os
+import networkit as nk
+import numpy as np
+import scipy
+
+class TestCliques(unittest.TestCase):
+
+	
+	def testMaxClique(self):
+		G = nk.readGraph("input/jazz2_directed.gml",nk.Format.GML)
+		G.removeSelfLoops()
+		#without callback, testing maxOnly, expecting single Clique
+		A1 = nk.clique.MaximalCliques(G, maximumOnly=True).run().getCliques()
+		self.assertEqual(len(A1), 1)
+		
+	def testMaxCliqueWithCallback(self):
+		G = nk.readGraph("input/jazz2_directed.gml",nk.Format.GML)
+		G.removeSelfLoops()
+		callback = lambda x,y : 1,2
+		A2 = nk.clique.MaximalCliques(G, callback).run().getCliques()
```

Comment:
This is not the correct way to call `MaximalCliques` with a callback function. The second parameter is a bool.